docs/contents/services.md

```diff
@@ -4,7 +4,7 @@ The following services are currently supported by ACK and for details see the [S
 
 |AWS Service|Status|Controller|
 |-----------|------|----------|
-|Amazon [API Gateway V2](https://aws.amazon.com/api-gateway/)|`BUILD`|`apigateway`|
+|Amazon [API Gateway V2](https://aws.amazon.com/api-gateway/)|`DEVELOPER PREVIEW`|`apigateway`|
```

Comment:
please "linkify" the controller name like the ECR and S3 ones

Reply:
I will. Thanks!